lib/mortgage_calculator/version.rb

```diff
@@ -1,7 +1,7 @@
 module MortgageCalculator
   module Version
     MAJOR = 1
-    MINOR = 8
+    MINOR = 9
     PATCH = 1
 
```

Comment:
I believe we should set the patch number back to 0 when incrementing a major/minor version number.